tuf/ngclient/_internal/trusted_metadata_set.py

```diff
@@ -153,6 +153,9 @@ def update_root(self, data: bytes) -> None:
         Raises:
             RepositoryError: Metadata failed to load or verify. The actual
                 error type and content will contain more details.
+
+        Returns:
+            Metadata[Root]: deserialized root metadata
```

Comment:
Let's not duplicate the type annotation. Maybe:

```suggestion
            Deserialized and verified root Metadata object
```

same comment on other docstrings